src/scss/utilities/_positioning.scss

```diff
@@ -0,0 +1,31 @@
+
+// Will push a flex item to the top
+.ec-push-t {
+  margin-bottom: auto;
+}
+
+// Will push a flex item to the bottom
+.ec-push-b {
+  margin-top: auto;
+}
+
+// Will push a flex item to the left
+.ec-push-l {
+  margin-right: auto;
+}
+
+// Will push a flex item to the right
+.ec-push-r {
+  margin-left: auto;
+}
+
+.ec-push-r-from-1024 {
+  @include media__from-1024 {
+    margin-left: auto;
+  }
+}
+
+.ec-m--auto {
```

Comment:
Why this class is not grouped together with other `ec-m--XX` classes?

Reply:
Because even though it adjusts the margins in reality is for positioning/center the item.